CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 
 The format is (loosely) based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## [3.0.0] - 2022-03-11
```

Comment:
Link needs to be added to bottom of CHANGELOG.

Reply:
I think I fixed this and did this right but I am not totally sure ...